tool/net/help.txt

```diff
@@ -534,6 +534,12 @@ HOOKS
           each time redbean accepts a new client connection. If it returns
           `true`, redbean will close the connection without calling fork.
 
+  OnLogLatency(reqtimems:int,contimems:int)
```

Comment:
If you're concerned about extending this API, then see third_party/lua/lunix.c. You can use the source code for the unix.Stat object where you have an extendable struct store the fields. Then you'd just call the method to get the latency number you want, rather than having to worry about adding more arguments. This technique is super fast.

Reply:
No, I'm not concerned about the API, just thinking about the name. If you're good with that, then I'm good too.